web-ui/src/pages/Clusters/ClusterDetail.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useMemo, useState, useEffect } from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { Space, Button, Modal, message, Table } from 'antd'
+import { ExclamationCircleOutlined } from '@ant-design/icons'
+import { useSessionStorageState } from 'ahooks'
+
+import { deleteCluster, getClusterTopo } from '../../utils/api'
+import { Root } from '../../components/Root'
+import { ICluster } from '.'
+
+export interface IClusterInstInfo {
+  id: string
+  role: string
+  host: string
+  ports: string
+  os_arch: string
+  status: string
+  data_dir: string
+  deploy_dir: string
+}
+
+export default function ClusterDetailPage() {
+  const navigate = useNavigate()
+  const [clustersList] = useSessionStorageState<ICluster[]>('clusters', [])
+  const { clusterName } = useParams()
+  const cluster = useMemo(
+    () => clustersList.find((el) => el.name === clusterName),
+    [clustersList, clusterName]
+  )
+  const [destroyingCluster, setDestroyingCluster] = useState(false)
+
+  const [clusterInstInfos, setClusterInstInfos] = useSessionStorageState<
+    IClusterInstInfo[]
+  >(`${clusterName}_cluster_topo`, [])
+
+  const [loadingTopo, setLoadingTopo] = useState(false)
+
+  const columns = useMemo(() => {
+    return [
+      'ID',
+      'Role',
+      'Host',
+      'Ports',
+      'OS_Arch',
+      'Status',
+      'Data_Dir',
+      'Deploy_Dir',
+    ].map((title) => ({
+      title,
+      key: title.toLowerCase(),
+      dataIndex: title.toLowerCase(),
+    }))
+  }, [])
+
+  function destroyCluster() {
+    setDestroyingCluster(true)
+    deleteCluster(clusterName).then(({ data, err }) => {
+      setDestroyingCluster(false)
+      if (data) {
+        message.success('销毁成功!')
```

Comment:
These messages should be in English, or we make them templates and keep translation files elsewhere